client/layout/guided-tours/config-elements.js

```diff
@@ -0,0 +1,309 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import {
+	find,
+	mapValues,
+	omit,
+	property,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import ExternalLink from 'components/external-link';
+import {
+	posToCss,
+	getStepPosition,
+	getValidatedArrowPosition,
+	targetForSlug
+} from './positioning';
+
+const contextTypes = Object.freeze( {
+	next: PropTypes.func.isRequired,
+	quit: PropTypes.func.isRequired,
+	isValid: PropTypes.func.isRequired,
+	tour: PropTypes.string.isRequired,
+	tourVersion: PropTypes.string.isRequired,
+	step: PropTypes.string.isRequired,
+} );
+
+export class Tour extends Component {
```

Comment:
Is there a reason to keep all the components in a single file, instead of organizing them in separate files for each component?